queue/queue.py

```diff
@@ -26,9 +26,22 @@ def dequeue(self):
   def len(self):
     return self.size
 
-  def len(self):
-    pass
+class Node:
+	def __init__(self, value=None, next_node=None):
```

Comment:
A Node should be not be created with a null value. Therefore, the constructor should look like this:

```
def __init__(self, value, next_node=None):
```